curriculum/challenges/_meta/top-learn-function-basics/meta.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "TOP Learn function basics",
+  "isUpcomingChange": true,
```

Comment:
Do we want this to be visible when it gets merged or keep it hidden for now?

Looks like a few of the other blocks are still hidden as well. Not sure if that's intentional or not.

Reply:
I think we can make a separate PR to sweep through those and audit as needed.

Reply:
Yep that is intentional, I want to release the javascript part all at once.